src/components/Modal.tsx

```diff
@@ -100,17 +103,22 @@ export function Modal(props: Props) {
                   </div>
                 </div>
 
-                <X
-                  className="cursor-pointer"
-                  onClick={() => props.onClose(false)}
-                />
+                {!props.disableClosing && (
+                  <X
```

Comment:
What do you think about changing this react-icons instead of react-feather?

Reply:
I am not sure I follow. React Icons is a meta package and standalone doesn't provide icons. It's just bundle of different vendors for icons, including Feather.